src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/Durable/ElasticSearch/ElasticSearchLogShipper.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Serilog.Core;
+using Serilog.Debugging;
+using Serilog.Sinks.Elasticsearch.Durable;
+
+namespace Serilog.Sinks.Elasticsearch.Durable
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class ElasticSearchLogShipper : LogShipper<List<string>>
```

Comment:
```suggestion
    public class ElasticsearchLogShipper : LogShipper<List<string>>
```
Convention is to use Elasticsearch instead of ElasticSearch